tests/webgpu_basic_rendering.cpp

```diff
@@ -365,6 +361,9 @@ int main() {
         run();
     });
 
-    // result will be reported when the main_loop completes
-    emscripten_exit_with_live_runtime();
+    // Result will be reported when the main_loop completes.
+    // emscripten_exit_with_live_runtime isn't needed because the WebGPU
+    // callbacks should all automatically keep the runtime alive until
+    // emscripten_set_main_loop, and that should keep it alive until
+    // emscripten_cancel_main_loop.
```

Comment:
I usually put `return 99` here to make sure the return code from main is not used.